api/README.md

```diff
@@ -81,6 +164,8 @@ NOTE: The full SHA of the fetched manifest is returned in the HTTP response head
 
     x-wpt-sha: abcdef0123456789abcdef0123456789abcdef01
 
+__Response format__
+
```

Comment:
This example JSON is missing several keys.

The high-level object has "items", "paths", "url_base", and "version".  Inside of "items", we have 8 keys: "manual", "reftest", "reftest_node", "stub", "support", "testharness", "visual", and "wdspec".  I *think* I can ignore reftest_node, stub, and support if I'm trying to just get at all the test paths?

Reply:
The docs here are limited deliberately since web-platform-tests/wpt would prefer to be able to freely change the manifest format and treat the python code as the source-of-truth and only handler.

Reply:
All right, in that case the important missing bit is that "visual" is a test type but isn't in the list here.

Reply:
Good spotting. https://github.com/web-platform-tests/wpt.fyi/pull/251